store/src/lora_valid_poc.rs

```diff
@@ -111,9 +126,13 @@ impl TryFrom<LoraValidWitnessReport> for LoraValidWitnessReportV1 {
     type Error = Error;
     fn try_from(v: LoraValidWitnessReport) -> Result<Self> {
         let report: LoraWitnessReportReqV1 = v.report.into();
+        let location = match v.location {
```

Comment:
Why is this still a String then?

Reply:
it is converted back to a string when going from internal struct to the proto def